plugins/__init__.py

```diff
@@ -1,12 +1,11 @@
 # -*- coding: utf-8 -*-
 
+from airflow.plugins_manager import AirflowPlugin
 # Inbuilt Imports
 from flask import Blueprint
-from airflow.plugins_manager import AirflowPlugin
-from airflow import configuration
```

Comment:
not needed? i dont recall what configuration was used for

Reply:
both imports weren't used in the file. Looks like usage was removed a while ago
https://github.com/mozilla/telemetry-airflow/commit/e1518a5ff5468bf1190cf5cb6c8142471d59c93e#diff-cbdca579e809f52334988bb9b86e40dee05abc7d11f5127923b1b20035a39772